config/codenarc/codenarc.xml

```diff
@@ -143,6 +143,7 @@
         <exclude name="UnnecessaryObjectReferences"/>
         <exclude name="UnnecessaryBigDecimalInstantiation"/>
         <exclude name="UnnecessarySetter"/>
+        <exclude name="UnnecessaryGString"/>
```

Comment:
I just got so sick of this.  As Java programmers writing occasional tests in Groovy, it seems unnecessary to enforce this constraint on us. Just allow the double quotes for simple strings and move on.